rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "1.70.0"
+components = ["rustfmt", "clippy", "rls", "rust-analysis"]
```

Comment:
Just learning Rust so I may be missing some history/BCP here, but [the `rustup` book says `rls` is deprecated and replaced by `rust-analyzer`](https://rust-lang.github.io/rustup/concepts/components.html#components); do we still need the old RLS?

Reply:
True, it's deprecated and we don't even use it.